saofile.js

```diff
@@ -24,7 +25,18 @@ module.exports = {
     const { cliOptions = {} } = this.sao.opts
     const edge = cliOptions.edge ? '-edge' : ''
 
+    const ifTrue = (condition, content, elseContent) => condition ? content : elseContent || ''
+    const generateComponent = (imports, componentOptions) => {
+      return `
+<script ${ifTrue(typescript, 'lang="ts"')}>
+${ifTrue(typescript, 'import Vue from \'vue\'')}
+${ifTrue(!!imports, imports)}
```

Comment:
I think that is there are imports, a carriage return should be ensured between imports and export.